### PR TITLE
Done

### PR DESCRIPTION
**Pull request by vovkandrew**
P.S. I have no idea why pom file was changed

### CODE REVIEW

File: src/main/java/core/basesyntax/SqrtOperation.java [hunk 1]
```diff
@@ -7,7 +7,11 @@
  * где вместо вопросика будет подставлено фактически переданное в метод число.</p>
  */
 public class SqrtOperation {
-    public double sqrt(double number) {
-        return 0.0;
+    public double sqrt(double number) throws IllegalArgumentException {
+        if (number >= 0) {
+            return Math.sqrt(number);
+        } else {
```
Comment (Tarabarnik) [inline src/main/java/core/basesyntax/SqrtOperation.java]: do we need this `else`?

Comment (vovkandrew) [reply src/main/java/core/basesyntax/SqrtOperation.java]: Fixed

**Comment by boroda4436** [comment]
@VovkAA Thavis check was failed. Please fix it

File: src/main/java/core/basesyntax/SqrtOperation.java [hunk 2]
```diff
@@ -7,7 +7,10 @@
  * где вместо вопросика будет подставлено фактически переданное в метод число.</p>
  */
 public class SqrtOperation {
-    public double sqrt(double number) {
-        return 0.0;
+    public double sqrt(double number) throws IllegalArgumentException {
```
Comment (Tarabarnik) [inline src/main/java/core/basesyntax/SqrtOperation.java]: why did you add throws here?

**Review by dmytro-hryhoruk — APPROVED** [review]
